test/upgrade/postdowngrade.go

```diff
@@ -28,3 +28,11 @@ func ChannelPostDowngradeTest() pkgupgrade.Operation {
 			runChannelSmokeTest(c.T)
 		})
 }
+
+// SourcePostDowngradeTest tests source operations after downgrade.
+func SourcePostDowngradeTest() pkgupgrade.Operation {
```

Comment:
I don't know but does it make sense to rename all these functions to have the `Event` prefix? 
E.g. EventSourceDowngradeTest

Reply:
This repo contains Kafka things. KafkaSource and KafkaChannel. I'm shorting Kafka, but not only I. That naming was already in this repo before. Let's not create new names like enentsource.

Reply:
This repo contains Kafka things. KafkaSource and KafkaChannel. I'm shorting Kafka, but not only I. That naming was already in this repo before. Let's not create new names like enentsource.

Reply:
It looks like these two (Source and KafkaSource) are totally mixed in function names in this repo. E.g. runSourceSmokeTest vs. AssureKafkaSourceIsOperational vs. testKafkaSource vs. SourcePostDowngradeTest
That's a cosmetic thing, having just "Source" is a bit more misleading to me. Nevermind